docs/404.html

```diff
@@ -1,149 +0,0 @@
-<!-- Generated by pkgdown: do not edit by hand -->
```

Comment:
everything in `docs/` is deleted, because that pollutes the repo. Instead we know publish from the `gh-pages` branch automatically on every commit.